<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockAFSK1200</class>
 <widget class="QFrame" name="DockAFSK1200">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AFSK1200 Decoder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/icons.qrc">
    <normaloff>:/icons/icons/terminal.svg</normaloff>:/icons/icons/terminal.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBar" name="toolBar">
     <property name="windowTitle">
      <string>toolBar</string>
     </property>
     <addaction name="actionClear"/>
     <addaction name="actionSave"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="cbEnabled">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="textView">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="maximumBlockCount">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="../../resources/icons.qrc">
     <normaloff>:/icons/icons/clear.svg</normaloff>:/icons/icons/clear.svg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear received packets</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources/icons.qrc">
     <normaloff>:/icons/icons/floppy.svg</normaloff>:/icons/icons/floppy.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save received packets to a text file</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="../../resources/icons.qrc">
     <normaloff>:/icons/icons/info.svg</normaloff>:/icons/icons/info.svg</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="toolTip">
    <string>Show info about AFKS1200 decoder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>on_cbEnabled_toggled(bool)</slot>
 </slots>
</ui>
